http-support/src/test/java/org/eclipse/sw360/antenna/http/HttpClientFactoryImplTest.java

```diff
@@ -74,4 +74,15 @@ public void testNewClientWithProxySettings() {
         assertThat(address.getPort()).isEqualTo(port);
         assertThat(proxy.type()).isEqualTo(Proxy.Type.HTTP);
     }
+
+    @Test
+    public void testNewClientWithoutCertificateCheck() {
+        System.setProperty(HttpClientFactoryImpl.CLIENT_ACCESS_UNVERIFIED_PROPERTY, "true");
```

Comment:
The system property should be unset after the test - best in a try-finally-block -; otherwise it might impact other tests, as this is a VM-wide setting.